components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -786,7 +786,7 @@ - (NSString *)accessibilityLabel {
   NSMutableArray *accessibilityStrings = [[NSMutableArray alloc] init];
   if ([super accessibilityLabel].length > 0) {
     [accessibilityStrings addObject:[super accessibilityLabel]];
-  } else if (self.placeholderLabel.accessibilityLabel.length > 0) {
+  } else if (!self.hidesPlaceholderOnInput && self.placeholderLabel.accessibilityLabel.length > 0) {
```

Comment:
What happens if hidesPlaceholderOnInput is true?

Reply:
If we are hiding it then we should not be adding it to the set of strings that are spoken.

we are only adding to the verbalized set if it is is visible.

Reply:
My expectation of the logic would then be:

```
if hidesPlaceholderOnInput {
  if hasInput {
    // Use input
  } else {
    // Use placeholder
  }
} else {
  // Use placeholder (because it's not hidden?)
}
```

Reply:
Is that accurate?

Reply:
im getting confused by the verbalization of voice over.

There appears to be a lower pitch verbalization of the placeholder text regardless of what we return from the accessibilityLabel

Reply:
your questions here made me rethink the upstream change. We need to change behavior: https://github.com/material-components/material-components-ios/pull/8976